### DISCUSSION TITLE
Autocomplete VS Code for Nuxt

### DISCUSSION THREAD

**Discussion by go4cas** [unocss/unocss#2638] [Q&A]
I have a Nuxt project, and using VS Code as the IDE. I also have the UnoCSS extension for VS Code installed. The project has a separate `uno.config.ts` file in the project root. I can't seem to get autocomplete working in my .vue file templates. Do I need to add any config settings?

**Comment by momoathome** [comment]
@go4cas see the latest Features, it is now Possible with the Setting "Fuzzy Search" in UnoCSS and VS Code

**Comment by zyyv** [comment]
Sounds no problem, as @momoathome say, try latest version again. If you have a minimal  reproduction repo is better.
